Standard/BisCustomAttributes.remarks.md

```diff
@@ -0,0 +1,56 @@
+---
+noEditThisPage: true
+remarksTarget: BisCustomAttributes.ecschema.md
+---
+
+# BisCustomAttributes
+
+## SchemaGradeValue

```

Comment:
I took a look at existing BIS schemas and enumerations are either postfixed with 'Type' or have no postfix.  For example SurfaceVariationType and CoordinateSystem from StructuralAnalysis or AutoHandledPropertyStatementType and ExternalSourceAttachmentRole.

For these enumerations I think 'SchemaGrade' and 'SchemaLayer' are good names because the enumerators are grades and layers respectively.  'Type' is a helpful postfix when the enumerators do not have another name.

We don't provide any guidance in the bis naming guidelines but I think we should add some: https://www.itwinjs.org/bis/naming-guidelines/rules-and-recommendations/